cli/server.ts

```diff
@@ -30,7 +30,7 @@ async function main() {
 
   app.use(cors())
 
-  app.use(bodyParser.urlencoded())
+  app.use(bodyParser.urlencoded({ extended: true }))
```

Comment:
what does this do?

Reply:
This was the default value, but with the current version if we don't specify it, it throws a warning message in the console.

Reply:
And even better:

```suggestion
  app.use(express.urlencoded({ extended: true }))
```

> This middleware is available in Express v4.16.0 onwards.

Source: https://expressjs.com/fr/api.html#express.urlencoded

Reply:
To be honest, I would like to skip the big lockfile upgrade. See the failed image builds caused by mocha and some other changed packages. Only stick to the minimum

Reply:
This change will not have any impact in the app, since in `package.json` it is using `"express": "^4.17.1"`, so it's already supported ;)

body-parser is deprecated for `urlencoded` and `json` middlewares, since they are now part of `express` directly since 4.16.0